websites/0-9/0gomovies/iframe.ts

```diff
@@ -0,0 +1,23 @@
+const iframe = new iFrame();
+
+let sendback: {
+	currTime: number;
+	duration: number;
+	paused: boolean;
+};
+
+function send(): void {
```

Comment:
Why ???? The func is useless.

Reply:
hehe. i just used existing presences' codes and made the main changes. dint bother on small detail.
ok wait will make the changes you requested.